Mathlib/CategoryTheory/Monoidal/Bimod.lean

```diff
@@ -860,25 +888,27 @@ theorem comp_whisker_left_bimod {W X Y Z : Mon_ C} (M : Bimod W X) (N : Bimod X
   rw [tensorRight_map]
   slice_rhs 1 3 => rw [π_tensor_id_preserves_coequalizer_inv_desc]
   slice_rhs 3 4 => rw [ι_colimMap, parallelPairHom_app_one]
-  slice_rhs 2 3 => rw [← id_tensor_comp, ι_colimMap, parallelPairHom_app_one]
+  slice_rhs 2 3 => rw [← MonoidalCategory.whiskerLeft_comp, ι_colimMap, parallelPairHom_app_one]
   slice_rhs 3 4 => rw [coequalizer.π_desc]
   dsimp [AssociatorBimod.invAux]
-  slice_rhs 2 2 => rw [id_tensor_comp]
+  slice_rhs 2 2 => rw [MonoidalCategory.whiskerLeft_comp]
   slice_rhs 3 5 => rw [id_tensor_π_preserves_coequalizer_inv_desc]
-  slice_rhs 2 3 => rw [associator_inv_naturality]
-  slice_rhs 1 3 => rw [Iso.hom_inv_id_assoc, MonoidalCategory.tensor_id]
-  slice_lhs 1 2 => rw [tensor_id_comp_id_tensor, ← id_tensor_comp_tensor_id]
+  slice_rhs 2 3 => rw [associator_inv_naturality_right]
+  slice_rhs 1 3 => rw [Iso.hom_inv_id_assoc]
+  slice_lhs 1 2 => rw [← whisker_exchange]
 set_option linter.uppercaseLean3 false in
-#align Bimod.comp_whisker_left_Bimod Bimod.comp_whisker_left_bimod
+#align Bimod.comp_whisker_left_Bimod Bimod.comp_whiskerLeft_bimod
 
-theorem comp_whisker_right_bimod {X Y Z : Mon_ C} {M N P : Bimod X Y} (f : M ⟶ N) (g : N ⟶ P)
-    (Q : Bimod Y Z) : tensorHom (f ≫ g) (𝟙 Q) = tensorHom f (𝟙 Q) ≫ tensorHom g (𝟙 Q) := by
-  rw [← tensor_comp, Category.comp_id]
+theorem comp_whiskerRight_bimod {X Y Z : Mon_ C} {M N P : Bimod X Y} (f : M ⟶ N) (g : N ⟶ P)
+    (Q : Bimod Y Z) : whiskerRight (f ≫ g) Q = whiskerRight f Q ≫ whiskerRight g Q := by
+  ext
+  apply Limits.coequalizer.hom_ext
+  simp
 set_option linter.uppercaseLean3 false in
-#align Bimod.comp_whisker_right_Bimod Bimod.comp_whisker_right_bimod
+#align Bimod.comp_whisker_right_Bimod Bimod.comp_whiskerRight_bimod
 
 theorem whisker_right_id_bimod {X Y : Mon_ C} {M N : Bimod X Y} (f : M ⟶ N) :
```

Comment:
```suggestion
theorem whiskerRight_id_bimod {X Y : Mon_ C} {M N : Bimod X Y} (f : M ⟶ N) :
```
In this file, there are a few other `whisker_left/right` names which should be changed similarly.